spec/jobs/appropriate_bodies/process_batch/action_job_spec.rb

```diff
@@ -33,7 +33,7 @@
           perform_action_job
         }.to have_broadcasted_to(
           "batch_progress_stream_#{pending_induction_submission_batch.id}"
-        ).from_channel(pending_induction_submission_batch).exactly(11).times
```

Comment:
`./spec/support/shared_contexts/csv_file.rb` contains mock CSV data for some specs including `fail` which it now shouldn't (or at least not claim that is a valid option), so we need to edit that.